app/controllers/user/audits_controller.rb

```diff
@@ -1,17 +1,8 @@
 # frozen_string_literal: true
 
-class User::AuditsController < User::BaseController
+class User::AuditsController < ApplicationController
   # GET
   def index
-    # create distinct list of
-    # - the audits of this user's actions
-    # - the actions anyone has taken on this user's shares
-    @audits = current_user.audits | shares_audits
-  end
-
-  private
-
-  def shares_audits
-    current_user.shares.map(&:audits).flatten
+    @audits = policy_scope(Audit).where(user: current_user).order(:created_at)
```

Comment:
⚠️ this changes the behaviour - it should also show audits for the user's shares (`Audit. for_user_shares(current_user)`)